apps/web/src/components/common/NameInput/index.tsx

```diff
@@ -12,25 +12,32 @@ const NameInput = ({
   name: string
   required?: boolean
 }) => {
-  const { register, formState } = useFormContext() || {}
+  const { register, formState, control } = useFormContext() || {}
   // the name can be a path: e.g. "owner.3.name"
   const fieldError = get(formState.errors, name) as FieldError | undefined
 
   return (
-    <TextField
-      {...props}
-      variant="outlined"
-      label={<>{fieldError?.type === 'maxLength' ? 'Maximum 50 symbols' : fieldError?.message || props.label}</>}
-      error={Boolean(fieldError)}
-      fullWidth
-      required={required}
-      className={inputCss.input}
-      onKeyDown={(e) => e.stopPropagation()}
-      {...register(name, {
-        maxLength: 50,
-        required,
-        setValueAs: (value) => value.trim(),
```

Comment:
This trim started to cause issues after I added the Controller. When typing something like `Safe name` it would change it to `Safename`. Not sure why it worked before but now it is possible again to add empty member names which we need to solve in a different way.